### ISSUE TITLE
Removed from Google Play Store

### ISSUE THREAD

**Issue by fpnewton** [daneren2005/Subsonic#1164]
Was this removed from the Google Play store? I can no longer install it after getting a new phone and all links to the Google Play store listing return a 404 now.

**Comment by FacialLeather** [comment]
I think it was removed a long time ago. This is the latest. 

https://github.com/daneren2005/Subsonic/releases

**Comment by daneren2005** [comment]
It has been removed because of
```
Your app is missing a Data safety form in the Play Console. All apps are required to complete an accurate Data safety form in Play Console that discloses their data collection and sharing practices - this is required even if your app does not collect any user data.
```

I got a couple of warning emails and tried to figure out what they were talking about but could never find anywhere that I needed to add a data safety form.  I had already filled the form out and they won't tell me what else I need to do still.  I just went in today and tried a few more things to see if that gets them to put the app back up or not.  I wasn't too worried about it before since they have basically de-listed it because I'm not jumping through hoops every year to do API updates anymore.  I didn't realize that they were going to completely remove it so people who previously payed for it couldn't re-download it anymore.  I will try to get it back up so at least people who previously paid can continue to access it.
